code/mob/living/silicon/robot.dm

```diff
@@ -1088,6 +1088,11 @@
 					src.part_chest?.cell = null
 
 	attackby(obj/item/W, mob/user)
+		if (istype(W, /obj/item/card/emag))
+			return
+		if (user.a_intent == INTENT_HARM)
+			..()
+			return
```

Comment:
This change will allow tool special attacks done on disarm also hit cyborgs (i.e. screwdriver multi-stab)

```suggestion
	attackby(obj/item/W, mob/user, params, is_special = 0, silent = FALSE)
		if (istype(W, /obj/item/card/emag))
			return
		if (user.a_intent == INTENT_HARM || is_special)
			..()
			return
```